src/lwc/geFormField/geFormField.js

```diff
@@ -330,28 +329,33 @@ export default class GeFormField extends LightningElement {
     load(data) {
         let value;
         if (data.hasOwnProperty(this.sourceFieldAPIName)) {
-            if (data[this.sourceFieldAPIName].hasOwnProperty('value')) {
-                value = data[this.sourceFieldAPIName].value;
+            value = data[this.sourceFieldAPIName];
+            if (value === null || value.value === null) {
+                this.reset();
             } else {
-                value = data[this.sourceFieldAPIName];
+                this.value = value.value || value;
```

Comment:
Yeah, part of the reason this is needed is because of how the lookup display values are saved on each row in the table and then need to be loaded into the form.  This should def be part of the larger convo about how the data structures should be composed for the load function.